boards/arm/nucleo_f429zi/nucleo_f429zi_defconfig

```diff
@@ -9,14 +9,23 @@ CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
 # Enable MPU
 CONFIG_STM32_ARM_MPU_ENABLE=y
 
+CONFIG_SERIAL=y
 # enable USART2 - passthrough to STLINK v2 connector
 CONFIG_UART_STM32_PORT_3=y
 
+# console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
 # enable pinmux
 CONFIG_PINMUX=y
 
 # enable GPIO ports A, B, C, D, G
 CONFIG_GPIO=y
+CONFIG_GPIO_STM32_PORTA=y
+CONFIG_GPIO_STM32_PORTB=y
+CONFIG_GPIO_STM32_PORTC=y
+CONFIG_GPIO_STM32_PORTD=y
```

Comment:
The comment says port G is also enabled, but that's not done here. Which should it be?

Reply:
It was not enabled before this patch. I checked boards pinmux file and port G is used for the ethernet, so I will also enable it.